### ISSUE TITLE
HTTPS location fails to add

### ISSUE THREAD

**Issue by zombor** [NewProggie/SparkleShare-Android#52]
I've got an https sparkleshare dashboard, and when I enter the url for it, it says "Sorry, but we weren't able to reach the host...".

This maybe be due to my self signed certificate, as it worked with normal http.

**Comment by ddeighton** [comment]
I'm having the same issue. I tried importing my self-signed cert into my phone, but that didn't help.

**Comment by monreal** [comment]
Maybe related, I am currently trying to get the android client to connect to a dashboard instance running behind apache (server.com/sparkle instead of server:3000) and this also doesn't seem to work. Maybe this stuff is hardcoded?

**Comment by NewProggie** [comment]
@zombor:
Can you give me the logcat output for a specific error message?

@monreal:
server port is not hardcoded. However the app assumes port 3000 if no port was given inside the url

**Comment by monreal** [comment]
Still not sure if my problem is maybe the same as zombor's. The fact that he is trying to use HTTPS means he also uses some other webserver as a reverse proxy? Or does the dashboard support HTTPS natively? 

When configuring the reverse proxy (on HTTP or HTTPS doesn't matter), the dashboard is no longer reachable on :3000 (or whatever port it is set to run). However, the android app still tries to connect to this port.

**Comment by zombor** [comment]
Yeah, I'm proxying behind apache. I'll try and get a log from the phone tomorrow.

**Comment by kaspro** [comment]
I'm having the same issue. proxy through apache, self-signed certificate. I installed my root-certificate to the device, without any luck. 
This seems to be an old issue - has it already been resolved?
Thanks in advance :)

**Comment by jschlatow** [comment]
I'm not using a proxy but HTTPS with a self-signed certificate. The android client claims the host that the host is not reachable, but with HTTP it works. I installed the certificate via Settings->Security->Install From SD but that did not change anything.
I wonder whether the client supports self-signed certificates at all?!

**Comment by comzeradd** [comment]
I used a certificate from StartSSL, which is an certified authority (at least for firefox) but still can't connect :/ Please fix this. Nobody wants to connect on unencrypted channel through port 3000.

**Comment by comzeradd** [comment]
It seems to work just fine. Just use:
https://sparkedomain:443
and it connects.

**Comment by ToniH1987** [comment]
Got the same Problem.
Sparkleshare behind SSL-Apache.
And just adding the Port-Number doen't work for me.
http://domain:443/sparkle won't work.

Any suggestions?

**Comment by comzeradd** [comment]
Try:
https://domain:443/sparkle

**Comment by ToniH1987** [comment]
My Bad.
I meant:
https://domain:443/sparkle

I'm familiar with linux, but not with andoird. Is there a way reading the logs without having root?

**Comment by Daniel-Marynicz** [comment]
Hello to fix this i create this pull request
https://github.com/NewProggie/SparkleShare-Android/pull/73

**Comment by Daniel-Marynicz** [comment]
And I upload SparkleShare-fix-issue-52.apk http://dl.dropbox.com/u/37563517/SparkleShare-fix-issue-52.apk
To fix this please  install  this apk.
 On Welcome Screen, Enter to options by menu key. And change option Accept all SSL certifcates to true.

**Comment by ToniH1987** [comment]
Perfekt! Works great!

Thank You!

**Comment by vauvenal5** [comment]
Update of the app in Google Play would be nice.
